src/Aspire.ProjectTemplates/templates/aspire-starter/9.2/Aspire-StarterApplication.1.AppHost/Program.cs

```diff
@@ -4,10 +4,12 @@
 var cache = builder.AddRedis("cache");
 
 #endif
-var apiService = builder.AddProject<Projects.GeneratedClassNamePrefix_ApiService>("apiservice");
+var apiService = builder.AddProject<Projects.GeneratedClassNamePrefix_ApiService>("apiservice")
+    .WithHttpsHealthCheck("/health");
```

Comment:
would adding a comment to explain what this is doing be overkill? ie "Telling Aspire dashboard to show health checks from '/health' endpoint"

Reply:
Seems like overkill, the method signature help tooltip thing should explain it well enough?